README.md

```diff
@@ -250,6 +250,7 @@ Key-value protocol was developed and is used to provide communication layer betw
       * ```__testcase_start``` - sent by DUT, test case start data.
       * ```__testcase_finish``` - sent by DUT, test case result.
       * ```__exit``` - sent by DUT, test suite execution finished.
+      * ```__exit_event_queue``` - sent by host test, indicating no more events expected.
```

Comment:
@mazimkhan I did not change the `__notify_complete` documentation. This is good to know though, it sounds like I can probably remove the extra `__exit_event_queue` event and just use `__notify_complete`. I will have to look at this again.